tools/runners/run-test-suite.sh

```diff
@@ -143,34 +147,34 @@ do
 
     full_test=$TESTS_DIR/${test#./}
 
+    comment=""
```

Comment:
move this into the else branch. that will also ensure that comment is set on all control paths and follows the logic of how cmd_line was set.